pkg/machine/wsl/machine.go

```diff
@@ -40,10 +40,12 @@ func (e *ExitCodeError) Error() string {
 	return fmt.Sprintf("Process failed with exit code: %d", e.code)
 }
 
+//nolint:unused
```

Comment:
@n1hility Can you take a look at this file and tell me if any of these are safe to remove? I kept them around assuming they might be used later.

Reply:
some of these will be needed for the port cleanup logic, so if you could leave them around a bit. I can clean them up as part of that work. thanks!